### PR TITLE
Feature/lyric feature gridsearch

### PR DESCRIPTION
**Pull request by Johny-Code**

### CODE REVIEW

File: utils/draw_plot.py
Comment (michsak) [inline utils/draw_plot.py:1]: I think this should be put in train_network file, moreover plotting acc loss already exists, you may modify it a little if needed add legend etc.

**Review by michsak — CHANGES_REQUESTED** [review]

File: utils/draw_plot.py
Comment (Johny-Code) [reply utils/draw_plot.py:1]: done

File: models/audio/train_resnet.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
 
 from implementation.ResNet18 import ResNet18
-from train_network import train_val_split, plot_acc_loss
+from utils.train_network import train_val_split
```
Comment (michsak) [inline models/audio/train_resnet.py]: na linuksie to tak nie dziala dlatego jest utils dodane. from draw_plot import plot_acc_loss

Comment (michsak) [reply models/audio/train_resnet.py]: and from train_network import train_val_split

**Review by michsak — APPROVED** [review]

**Pull request merged**